.github/workflows/lint-docs-weekly.yml

```diff
@@ -2,6 +2,7 @@ name: Weekly check all Markdown links
 
 on:
   schedule:
+    # Cron for every Thursday at 12:00 UTC.
```

Comment:
Q: should this be run at the same time as trivy?

Can you also add a similar comment to scan.yml please